src/components/Buttons/ProfileButton/index.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from "prop-types"
+
+import * as S from "./styles"
+
+const ProfileButton = (props) => {
+  return (
+    <S.Container>
+      <S.PhotoContainer>
+        <S.Photo
+          src={props.imgUrl}
+          alt="user-photo"
+        />
+      </S.PhotoContainer>
+      <S.Name>{props.name}</S.Name>
+      <S.Name>{props.name}</S.Name>
+    </S.Container>
+  )
+}
+
+// ProfileButton.propTypes = {
+//   imgUrl: PropTypes.string.isRequired,
+//   name: PropTypes.string.isRequired
+// }
```

Comment:
Uma vez que os proptypes estão corretos, tu pode descomentar isso aqui.
```suggestion
ProfileButton.propTypes = {
  imgUrl: PropTypes.string.isRequired,
  name: PropTypes.string.isRequired
}
```